hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/CryptoGetAccountInfoHandlerTest.java

```diff
@@ -540,7 +540,6 @@ private AccountInfo getExpectedAccountInfoEvm(Bytes evmAddress) {
                 .ownedNfts(2)
                 .maxAutomaticTokenAssociations(10)
                 .ethereumNonce(0)
-                .alias(evmAddress)
```

Comment:
Why did we remove this ?

Reply:
To match [mono](https://github.com/hashgraph/hedera-services/blob/27fe1a07dabdf6999015812505acbb78de753032/hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/context/primitives/StateView.java#L421), also pass HapiTest, I added !isOfEvmAddressSize(account.getAlias()) check in CryptoGetAccountInfoHandler to set alias. In this test, the alias is not set because !isOfEvmAddressSize(account.getAlias() is true.